token/cli/src/main.rs

```diff
@@ -2589,7 +2614,7 @@ fn app<'a, 'b>(
                         .possible_values(&[
                             "mint", "freeze", "owner", "close",
                             "close-mint", "transfer-fee-config", "withheld-withdraw",
-                            "interest-rate",
+                            "interest-rate","permanent-delegate",
```

Comment:
nit that I'm surprised didn't get caught by rustfmt
```suggestion
                            "interest-rate", "permanent-delegate",
```